### PR TITLE
Update restConnector features for mpOpenApi-3.0

### LINKED ISSUE #19796: Ensure restConnector-2.0 works with mpOpenApi-3.0
We have some integration between restConnector-2.0 and mpOpenApi to expose static OpenAPI documents for the rest APIs provided by restConnector-2.0.

It doesn't look like this was updated to work with mpOpenApi-3.0. It might be as straightforward as updating the autofeatures to also work with mpOpenApi-3.0, but we might also need to transform the bundles, which would mean new autofeatures.

### PR DESCRIPTION
**Pull request by Azquelt**
Make the OpenAPI documentation for the restconnector config and validation endpoints work with mpOpenApi-3.0.

This was missed during the development of mpOpenApi-3.0.

Fixes #19796

### CODE REVIEW

**Comment by Azquelt** [comment]
#build

**Review by nmittles — APPROVED** [review]
The changes look good to me.  My only comment is for updating copyright years for the following:

com.ibm.ws.rest.handler.config_fat/bnd.bnd
com.ibm.ws.rest.handler.config_fat/fat/src/com/ibm/ws/rest/handler/config/fat/ConfigOpenApiSchemaTest.java
com.ibm.ws.rest.handler.config_fat/fat/src/com/ibm/ws/rest/handler/config/fat/ConfigRESTHandlerJCATest.java
com.ibm.ws.rest.handler.validator_fat/fat/src/com/ibm/ws/rest/handler/validator/fat/ValidateDataSourceTest.java
com.ibm.ws.rest.handler.validator_fat/fat/src/com/ibm/ws/rest/handler/validator/fat/ValidateJCATest.java com.ibm.ws.rest.handler.validator_fat/fat/src/com/ibm/ws/rest/handler/validator/fat/ValidateJMSTest.java
io.openliberty.rest.handler.config.openapi.common/bnd.bnd
io.openliberty.rest.handler.validator.openapi.2.0/bnd.bnd

**Comment by Azquelt** [comment]
I didn't update the copyright date on the files where I'd only made very minor changes.

Looking over the files mentioned though, I should have updated it on ValidateJCETest and VaildateDataSourceTest.

**Comment by Azquelt** [comment]
#libby
#build

File: dev/com.ibm.ws.rest.handler.validator_fat/fat/src/com/ibm/ws/rest/handler/validator/fat/ValidateJMSTest.java [hunk 1]
```diff
@@ -10,7 +10,8 @@
  *******************************************************************************/
 package com.ibm.ws.rest.handler.validator.fat;
 
-import static componenttest.annotation.SkipForRepeat.EE9_FEATURES;
+import static com.ibm.ws.rest.handler.validator.fat.FATSuite.assertClassEquals;
```
Comment (tevans78) [inline dev/com.ibm.ws.rest.handler.validator_fat/fat/src/com/ibm/ws/rest/handler/validator/fat/ValidateJMSTest.java:13]: Copyright date

File: dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.configValidationConfigSchema-3.0.feature [hunk 2]
```diff
@@ -0,0 +1,12 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.configValidationConfigSchema-3.0
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.configValidationConfigSchema-3.0.feature]: Consider naming this `io.openliberty.configValidationConfigSchema-3.0`

File: dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.configValidationValidatorSchema-3.0.feature [hunk 3]
```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.configValidationValidatorSchema-3.0
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.configValidationValidatorSchema-3.0.feature]: Consider naming this `io.openliberty.configValidationValidatorSchema-3.0`

File: dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.configValidationConfigSchema-3.0.feature (hunk 2 above)
Comment (tevans78) [reply dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.configValidationConfigSchema-3.0.feature]: All new features should follow the `io.openliberty...` naming scheme

File: dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.configValidationValidatorSchema-3.0.feature (hunk 3 above)
Comment (tevans78) [reply dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.configValidationValidatorSchema-3.0.feature]: All new features should follow the `io.openliberty...` naming scheme

**Review by tevans78 — CHANGES_REQUESTED** [review]

File: dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.configValidationConfigSchema-3.0.feature (hunk 2 above)
Comment (Azquelt) [reply dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.configValidationConfigSchema-3.0.feature]: Thanks, I always forget that when I start by copying an existing feature :(

**Comment by Azquelt** [comment]
Updated feature names.
#build
#libby

**Review by tevans78 — APPROVED** [review]

**Pull request merged**